package.json

```diff
@@ -23,11 +23,12 @@
     "start": "react-native-scripts start",
     "android": "react-native-scripts android",
     "ios": "react-native-scripts ios",
+    "devtools": "react-devtools",
```

Comment:
@Knorrke starts the React Devtools. You can inspect components with that and change styles. Also nice: enabling remote debugging so that you see logs in browser console instead of having to `console.warn` (and allows you to set brekapoints)